gizmo.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 13 12:26:42 2021
+
+@author: arsii
+"""
+
+import numpy as np
+
+class Gizmo:
+    def __init__(self,name):
+        self.name = name
+        
+    def speak(self):
+        print(f'{self.name}')
+    
+    def multiplication_table(self, zero_out_multiples = None):
+        a = np.arange(1,13)
+        mm = np.outer(a,a)
+        if isinstance(zero_out_multiples, int):
+            for i in range(1,13):
```

Comment:
there's no need to do this row-by-row or column-by-column. Take a look at what `mm % zero_out_multiples == 0` gives you.